server/model/reviewAndSubmitAdaViewModel.ts

```diff
@@ -31,14 +31,17 @@ export default class ReviewAndSubmitAdaViewModel {
               { text: it.days, format: 'numeric' },
             ]
           }),
-          [
+        ],
+        summaryRow: {
+          row: [
             { html: '<strong>Total ADAs taken into calculation</strong>', colspan: 2 },
             {
               text: this.adjustments.map(a => a.days).reduce((sum, current) => sum + current, 0),
               format: 'numeric',
             },
           ],
-        ],
+          role: 'presentation',
```

Comment:
I think you can change this to 
```
attributes: {
  role: 'presentation'
}
```
and then you can keep the use of the govukTable  in the review and submit view and there is no need to maintain the whole table

Reply:
role required at row level instead of cell level so this doesn't suggestion doesn't work